test/integration/utils.go

```diff
@@ -143,3 +154,102 @@ func (e *eqWrapper) Used(used v1.ResourceList) *eqWrapper {
 func (e *eqWrapper) Obj() *v1alpha1.ElasticQuota {
 	return e.ElasticQuota
 }
+
+type testContext struct {
+	ClientSet          clientset.Interface
+	KubeConfig         *restclient.Config
+	InformerFactory    informers.SharedInformerFactory
+	DynInformerFactory dynamicinformer.DynamicSharedInformerFactory
+	Scheduler          *scheduler.Scheduler
+	Ctx                context.Context
+	CancelFn           context.CancelFunc
+}
+
+func initTestSchedulerWithOptions(t *testing.T, testCtx *testContext, opts ...scheduler.Option) *testContext {
+	testCtx.InformerFactory = scheduler.NewInformerFactory(testCtx.ClientSet, 0)
+	if testCtx.KubeConfig != nil {
+		dynClient := dynamic.NewForConfigOrDie(testCtx.KubeConfig)
+		testCtx.DynInformerFactory = dynamicinformer.NewFilteredDynamicSharedInformerFactory(dynClient, 0, v1.NamespaceAll, nil)
+	}
+
+	var err error
+	eventBroadcaster := events.NewBroadcaster(&events.EventSinkImpl{
+		Interface: testCtx.ClientSet.EventsV1(),
+	})
+
+	opts = append(opts, scheduler.WithKubeConfig(testCtx.KubeConfig))
+	testCtx.Scheduler, err = scheduler.New(
+		testCtx.ClientSet,
+		testCtx.InformerFactory,
+		testCtx.DynInformerFactory,
+		profile.NewRecorderFactory(eventBroadcaster),
+		testCtx.Ctx.Done(),
+		opts...,
+	)
+
+	if err != nil {
+		t.Fatalf("Couldn't create scheduler: %v", err)
+	}
+
+	stopCh := make(chan struct{})
+	eventBroadcaster.StartRecordingToSink(stopCh)
+
+	return testCtx
+}
+
+func syncInformerFactory(testCtx *testContext) {
+	testCtx.InformerFactory.Start(testCtx.Ctx.Done())
+	if testCtx.DynInformerFactory != nil {
+		testCtx.DynInformerFactory.Start(testCtx.Ctx.Done())
+	}
+	testCtx.InformerFactory.WaitForCacheSync(testCtx.Ctx.Done())
+	if testCtx.DynInformerFactory != nil {
+		testCtx.DynInformerFactory.WaitForCacheSync(testCtx.Ctx.Done())
+	}
+}
+
+func cleanupPods(t *testing.T, testCtx *testContext, pods []*v1.Pod) {
```

Comment:
Same here, any particular reason to duplicate the code here instead of reusing the k/k version?